src/components/DayPickerRangeController.jsx

```diff
@@ -174,6 +183,22 @@ export default class DayPickerRangeController extends React.Component {
     this.setDayPickerRef = this.setDayPickerRef.bind(this);
   }
 
+  componentDidMount() {
+    const { focusedInput } = this.props;
+    const { phrases } = this.state;
+
+    // initialize phrases
+    // set the appropriate CalendarDay phrase based on focusedInput
+    const chooseAvailableDate = getChooseAvailableDatePhrase(phrases, focusedInput);
```

Comment:
It seemed like the chooseAvailableDate phrase was only getting set on update of the focus or the phrases so I thought adding the setting of it in `componentDidMount` would address initialization.